neurolib/models/multimodel/builder/adex.py

```diff
@@ -0,0 +1,944 @@
+"""
+Adaptive exponential integrate-and-fire mean-field approximation.
+
+Main reference:
+    Augustin, M., Ladenbauer, J., Baumann, F., & Obermayer, K. (2017).
+    Low-dimensional spike rate models derived from networks of adaptive
+    integrate-and-fire neurons: comparison and implementation. PLoS Comput Biol,
+    13(6), e1005545.
+
+Additional reference:
+    Cakan C., Obermayer K. (2020). Biophysically grounded mean-field models of
+    neural populations under electrical stimulation. PLoS Comput Biol, 16(4),
+    e1007822.
+"""
+
+import logging
+import os
+from copy import deepcopy
+
+import numba
+import numpy as np
+import symengine as se
+from h5py import File
+from jitcdde import input as system_input
+
+from ..builder.base.constants import EXC, INH, LAMBDA_SPEED
+from ..builder.base.network import Network, SingleCouplingExcitatoryInhibitoryNode
+from ..builder.base.neural_mass import NeuralMass
+
+DEFAULT_CASCADE_FILENAME = "quantities_cascade.h5"
+
+DEFAULT_PARAMS_EXC = {
```

Comment:
A lot of these parameters have different names than in the original `aln` model. This could be pretty confusing, no? Why not load parameters from `loadDefaultParams.py`?

Reply:
the set of parameters in `loadDefaultParams.py` is just too different. The parameters in the original ALN model implementation contains all parameters even for the connectivity, structural matrix, all the delays, input noise parameters etc. The `MultiModel` design somehow automatically in a natural manner divides the parameter "groups" into parameters for the individual mass, then connectivity / delay between masses when building a node and then connectivity / delays when building a network out of nodes... I feel that loading parameters from this file would be more confusing since then I would need to divide parameters from one dictionary into two dictionaries (one for EXC, one for INH), then make a square matrix out of local connectivity and delays (`c`s and `d`s) etc...

as for the names, I would consolidate all the names in the separate issue for all `MultiModel` models depending on our agreement

Reply:
I actually mean the ones that are already here: 
# number of inputs per neuron from EXC/INH
```
    "K_exc": 800.0,
    "K_inh": 200.0,
    # postsynaptic potential amplitude for global connectome
    "c_global": 0.4,
    # number of incoming EXC connections (to EXC population) from each area
    "K_exc_global": 250.0,
    # synaptic time constant EXC/INH
    "tau_syn_exc": 2.0,  # ms
    "tau_syn_inh": 5.0,  # ms
    # internal Fokker-Planck noise due to random coupling
    "sigma_ext": 1.5,  # mV/sqrt(ms)
    # maximum synaptic current between EXC/INH nodes in mV/ms
    "J_exc_max": 2.43,
    "J_inh_max": -3.3,
```

Many of these are the same as in the `aln` model but have different names.